lib/model/migrations/20250113-01-disable-nullable-blob-content-types.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = (db) => db.raw(`
+  ALTER TABLE blobs
+    ALTER COLUMN "contentType" SET NOT NULL,
+    ALTER COLUMN "contentType" SET DEFAULT 'application/octet-stream'
+`);
```

Comment:
When I run this (with null content type values in my DB) I get an error

```
migration file "20250113-01-disable-nullable-blob-content-types.js" failed
migration failed with error:
  ALTER TABLE blobs
    ALTER COLUMN "contentType" SET NOT NULL,
    ALTER COLUMN "contentType" SET DEFAULT 'application/octet-stream'
 - column "contentType" of relation "blobs" contains null values
 ```

Reply:
:facepalm: seems like a good candidate for a test :grin:

Reply:
Test added.